gulpfile.js

```diff
@@ -487,19 +493,28 @@ gulp.task('release-dist', ['build-dist'], function(done) {
         if (platformIsActive(osArch)) {
             switch (osArch) {
                 case 'win-ia32':
-                    shell.cp(path.join(distPath, 'win-ia32', `${applicationName} Setup ${version}-ia32.exe`), releasePath);
+                    shell.cp(path.join(distPath, 'win-ia32', `${applicationName} Setup ${version}-ia32.exe`),
+                            path.join(releasePath, `${applicationName.replace(/\s/, '-')}-win32-${version.replace(/\./g, '-')}.exe`));
```

Comment:
This code - `version.replace(/\./g, '-')` - should be calculated once and then re-used. This will increase code clarity.